src/media/js/views/search.js

```diff
@@ -194,11 +149,6 @@ define('views/search',
                     data.unshift(base('Wa-pa-pa-pa-pa-pa-pow!'));
                     data.unshift(base('Ring-ding-ding-ding-dingeringeding!'));
                     break;
-                case 'hampster dance':
-                    data.forEach(function(v, k) {
-                        v.icons['64'] = urls.media('fireplace/img/icons/eggs/h' + (k % 4 + 1) + '.gif');
-                    });
-                    break;
                 case 'rick fant rolled':
```

Comment:
Got rid of these since we nuked the image assets for them anyway. Having some easter eggs shouldn't be too bad.